src/browser/browser.js

```diff
@@ -210,12 +211,14 @@ define((require, exports, module) => {
                }
               }),
 
-      WebViewer.Deck({key: 'web-viewers',
-                      className: 'iframes',
-                      items: webViewersCursor,
-                      onClose: item => webViewersCursor.update(closeTab(item)),
-                      onOpen: item => webViewersCursor.update(addTab(item))
-                     })
+      WebViewer.Deck({
+        key: 'web-viewers',
```

Comment:
Comment: FWIW, I prefer this style of indentation in C-style languages because it less frequently runs up against the 80 char limit.